cmd/main.go

```diff
@@ -18,11 +18,9 @@ func init() {
 }
 
 func main() {
-	l := slog.New(slog.NewJSONHandler(os.Stdout, nil))
-
 	defer func() {
 		if x := recover(); x != nil {
-			l.Error(string(debug.Stack()))
+			slog.Error(string(debug.Stack()))
```

Comment:
This is the default slog rather than the custom one used in main. Therefore, the log is not in json format.
I decided that there is no problem even if the error that occurs in main is not in json format.
The implementation on the proxy side sets a custom slog.